services/minespace-web/src/components/pages/Project/InformationRequirementsTablePage.js

```diff
@@ -143,17 +143,22 @@ const StepForms = (
         type="tertiary"
         className="full-mobile"
         onClick={() => {
-          prev();
-          props.history.push(
-            routes.ADD_INFORMATION_REQUIREMENTS_TABLE.dynamicRoute(props.project.project_guid)
-          );
+          props.history.push({
+            pathname: `${routes.ADD_INFORMATION_REQUIREMENTS_TABLE.dynamicRoute(
+              props.project.project_guid
+            )}`,
+            state: { current: 1 },
+          });
         }}
-        disabled={state.submitting}
+        disabled={props.project?.information_requirements_table?.status_code === "APV"}
```

Comment:
should this also have the submitting flag?

Reply:
We disabled if the application is Approved. Otherwise, we allow to go back in resubmit an IRT.